src/ai2_internal/citation_links/interface.py

```diff
@@ -65,9 +66,15 @@ def predict_one(self, inst: Instance) -> Prediction:
         doc.annotate(mentions=[sg.to_mmda() for sg in inst.mentions])
         doc.annotate(bibs=[sg.to_mmda() for sg in inst.bibs])
 
-        prediction = self._predictor.predict(doc) # returns (mention.id, bib.id)
-        
-        return Prediction(linked_mentions = prediction)
+        prediction = self._predictor.predict(doc) # returns [(mention.id, bib.id)]
+
+        return Prediction(
+            linked_mentions = prediction,
+            linked_mention_relations = [
+                api.Relation(from_id=mention_id, to_id=bib_id)
```

Comment:
I thought we'd have to include the from and to annotation names at the timo interface ("mentions", "bib_entries")

Reply:
I discussed this at length with Tyler and Chloe a couple days ago because they had comments on my SPP PR. I think we decided the TIMO endpoint would just have a named field with no special designation beyond that.